.github/ISSUE_TEMPLATE/user-interface-or-usability-bug-report.md

```diff
@@ -2,7 +2,7 @@
 name: User Interface or Usability Bug report
 about: Please include screenshots in UI/UX bug reports
 title: ''
-labels: bug, ui/ux
+labels: T-Defect
```

Comment:
Just a thought, might not make sense. Since there is no `ux/ui` does it make sense to have a separate issue template for UI/UX?

Reply:
Ah yeah, good point. I'll fold the screenshot request into the general template, seems fine to at least ask for all cases.

Reply:
wait, but Z-UI/UX is still a thing?

https://github.com/vector-im/element-web/issues/11295

Reply:
Ah yeah, that one is in a kind of holding pattern... I'm currently tempted to delete it, but waiting on confirmation from various parties to check whether they'd be sad about that.

Reply:
I have gotten enough confirmation that it's effectively unused, so `Z-UI/UX` should disappear soonish.